### DISCUSSION TITLE
2011/07/17/%E8%B1%86%E8%B1%86%E5%8F%AB%E7%88%B8%E7%88%B8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#340] [Announcements]
# 2011/07/17/%E8%B1%86%E8%B1%86%E5%8F%AB%E7%88%B8%E7%88%B8

虽然豆豆早就能发出“爸”这个音了，但他一直不太爱说这个字。直到最近一个星期，他突然就喜欢上叫“爸爸”了，时不时就叫几声。难道是因为我最近哄他玩的很开心？豆豆也能发“妈”的音了，但也几乎不怎么说。周末我又心血来潮，让豆豆叫“Daddy”，他居然就daddy, daddy的叫起来了。豆豆妈在一旁嫉妒的不得了。

https://blog.qizhen.xyz/2011/07/17/%E8%B1%86%E8%B1%86%E5%8F%AB%E7%88%B8%E7%88%B8

**Comment by ruanqizhen** [comment]
之前博客下的留言：


ronin
2011年07月17日 22:52 编辑
独辟蹊径啊，哈哈

回复
